frontend/packages/dev-console/src/components/add/AddCard.tsx

```diff
@@ -4,18 +4,19 @@ import { ResolvedExtension, AddAction } from '@console/dynamic-plugin-sdk';
 import AddCardItem from './AddCardItem';
 import './AddCard.scss';
 
-type AddCardProps = { title: string; items: ResolvedExtension<AddAction>[]; namespace: string };
-const AddCard: React.FC<AddCardProps> = ({ title, items, namespace }) => {
+type AddCardProps = {
+  id: string;
+  title: string;
+  items: ResolvedExtension<AddAction>[];
+  namespace: string;
+};
+
+const AddCard: React.FC<AddCardProps> = ({ id, title, items, namespace }) => {
   const isTitleFromItem: boolean = items?.length === 1 && items[0].properties.label === title;
   return items?.length > 0 ? (
-    <Card key={title} className="odc-add-card">
+    <Card key={title} className="odc-add-card" data-test={`card ${id}`}>
```

Comment:
This dynamic ID could clash the static `card getting-started` ID within the same scope.

Reply:
I think that's a feature, not a bug. If someone creates a dynamic "Getting Started" card, it's okay that our **tests** fail.

"For the tests" (visual) this is a card.

Reply:
But add-actions extensions and the getting start card are 2 separate contributed items. One is static and one is dynamic. And there is no scope difference. So while each card is visually distinguishable and valid, from a test perspective the test would need to change and not the implementation of the contributions.

Reply:
Removed the card from the `getting-started` (data-test) card.